### DISCUSSION TITLE
蓝灯有没有适用于iPad的版本

### DISCUSSION THREAD

**Discussion by GithubLexi666** [getlantern/lantern-forum-cn#443] [Q&A]
请问蓝灯有ios版本吗，目前没有找到，有的话可以留一个网址给我吗？

**Comment by GithubLexi666** [comment]
@Mike777Khoo

**Reply by MaxMKMax** [reply]
@GithubLexi666 
您好. 有的. 以下链接可以下载.
https://lantern.io/zh-Hans/download?os=ios


我们发布了新的 Flutter 桌面应用程序测试版（8.0.0 版本）。

Mac - https://github.com/getlantern/lantern-binaries/raw/main/lantern-installer-preview.dmg

Windows - https://github.com/getlantern/lantern-binaries/raw/main/lantern-installer-preview.exe

Linux - https://github.com/getlantern/lantern-binaries/raw/main/lantern-installer-preview-64-bit.deb

**Reply by GithubLexi666** [reply]
谢谢你Mike

**Reply by MaxMKMax** [reply]
@GithubLexi666 您好. 不客气! 蓝灯非常感谢您支持我们. 很开心有像您这么尊贵的用户.
